domenetjenester/brevbestiller/src/main/java/no/nav/foreldrepenger/melding/brevmapper/brev/opphørSVP/OpphørPeriodeMerger.java

```diff
@@ -0,0 +1,77 @@
+package no.nav.foreldrepenger.melding.brevmapper.brev.opphørSVP;
+
+import no.nav.foreldrepenger.melding.brevmapper.brev.felles.DatoVerktøy;
+import no.nav.foreldrepenger.melding.geografisk.Språkkode;
+import no.nav.foreldrepenger.melding.integrasjon.dokgen.dto.opphørSVP.OpphørPeriode;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class OpphørPeriodeMerger {
+    public static List<OpphørPeriode> mergePerioder(List<OpphørPeriode> opphørtePerioder, Språkkode språkkode) {
```

Comment:
Du slipper å ta inn og videresende Språkkode som argument hele veien hvis du lagrer den med JsonIgnore i DTOen, se eksempel på Utbetalingsperiode / UtbetalingsperiodeMerger for innvilgelse FP

Reply:
utført